android/README.md

```diff
@@ -0,0 +1,82 @@
+# IronOxide-Android
+
+[![javadoc.io](https://javadoc.io/badge2/com.ironcorelabs/ironoxide-java/javadoc.io.svg)](https://javadoc.io/doc/com.ironcorelabs/ironoxide-java)
+
+## Using IronOxide-Android
+
+In order to make calls to the IronCore service, you must add a dependency for ironoxide-android, load the library, and provide authentication.
+
+### Adding a dependency
+
+IronOxide-Android is published to a custom maven repository. To pull from this repository, add the following to your project's "build.gradle":
+
+```java
+allprojects {
+    repositories {
+        maven {
+            url "https://oss.sonatype.org/content/repositories/snapshots"
+        }
+    }
+}
+```
+
+Then add the following to your module's "build.gradle":
+
+```java
+dependencies {
+    implementation 'com.ironcorelabs:ironoxide-android:<VERSION>@aar'
+}
+```
+
+### Loading the library
+
+To load the `ironoxide-android` library in Java, use the following:
+
+```java
+java.lang.System.loadLibrary("ironoxide_android");
+```
+
+### Authenticating
+
+To make calls, you must create a project and segment in the IronCore Admin Console. Once created, you can use a tool such as [ironoxide-cli](https://github.com/IronCoreLabs/ironoxide-cli) to create users and devices. You can then read in a device with the `DeviceContext.fromJsonString()` function and use it to initialize an IronOxide instance. An example of this is available in the [Example Application](/android/examples/Example_Application/) in "MainActivity.java".
+
+## Build from Source
+
+### Prerequisites
+
+- Install [Rust toolchain](https://www.rust-lang.org/tools/install).
+- Install [cross](https://github.com/rust-embedded/cross).
+- Install Android SDK 29. You can get the command line SDK [here](https://developer.android.com/studio) (scroll down to "Command line tools only").
+  - Note: **The extracted `tools` folder must follow a specific folder hierarchy. We recommend `AndroidCLI/cmdline-tools/tools`**.
+- Install Android 29 build and platform tools. This can be done with `sdkmanager` (found in `tools/bin`):
+
+  ```bash
+  ./sdkmanager "build-tools;29.0.3" "platform-tools" "platforms;android-29"
+  ```
+
+- Create the file `$HOME/.gradle/gradle.properties` and add the line `sdk.dir=PATH_TO_ANDROID_CLI_FOLDER`.
+
+### Building
+
+From the repository root, run `android/build.sh`. This will put the compiled library and generated Java files into `android/ironoxide-android/src/main`.
+
+## Testing
+
+### Prerequisites
+
+- Successfully run `build.sh` by following the steps in [Build from Source](#build-from-source-1).
+  - This will create the java files and the `.so` files required for the `x86`, `x86_64`, and `arm64-v8a` architectures. If testing on a different architecture, you can find the Rust target to compile to [here](https://forge.rust-lang.org/release/platform-support.html).
```

Comment:
This is what `build-aar.sh` does now, right? `Build.sh` only builds one architecture.

Reply:
True. I think most people will only care about the x86_64 build, so I'll keep it as `build.sh` and change the wording